src/main/scala/ai/privado/languageEngine/ruby/passes/config/RubyPropertyLinkerPass.scala

```diff
@@ -0,0 +1,42 @@
+package ai.privado.languageEngine.ruby.passes.config
+
+import ai.privado.languageEngine.java.language.NodeStarters
+import ai.privado.tagger.PrivadoParallelCpgPass
+import io.shiftleft.codepropertygraph.generated.{Cpg, EdgeTypes}
+import io.shiftleft.codepropertygraph.generated.nodes.{AstNode, JavaProperty}
+import org.slf4j.LoggerFactory
+import io.shiftleft.semanticcpg.language.*
+
+class RubyPropertyLinkerPass(cpg: Cpg) extends PrivadoParallelCpgPass[JavaProperty](cpg) {
+
+  private val logger = LoggerFactory.getLogger(getClass)
+
+  val dbConnectionRegex =
+    "^(db|database|jdbc|mysql|postgres|oracle|sqlserver)_(connection_)?(host|port|name|user|password|uri|driver|ssl|pool_size|timeout|connection_string)$"
```

Comment:
Pick up the regex from privado config, don't hard code here

Reply:
This regex is not needed here. i'm removing it.